calico_versioned_docs/version-3.27/releases.json

```diff
@@ -1,4 +1,59 @@
 [
+  {
+    "title": "v3.27.4",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "registry": "quay.io",
+      "version": "v1.32.7"
```

Comment:
We'll need to update this. Not sure if v1.32.9 (latest v1.32) or if we'll need a new operator patch release, @danudey or @lwr20 would you know that?

Reply:
It'll be a new operator patch release.  Sounds like it will be 1.32.10 if .9 already exists